lib/built_redux_test_utils.dart

```diff
@@ -7,13 +7,13 @@ import 'built_redux.dart';
 /// It takes all of the same optional params as expectAsync.
 void expectDispatched<T>(
   ActionDispatcher<T> actionDispatcher, {
-  void verfier(Action<T> action),
+  void verifier(Action<T> action),
```

Comment:
This is a breaking change, so we'll need to release a V2 method to abide by our semver policy and avoid breaking consumers.

Reply:
i'm thinking
```dart
void expectDispatched<T>(
  ActionDispatcher<T> actionDispatcher, {
  void verifier(Action<T> action),
  @deprecated void verfier(Action<T> action),
  int count: 1,
  int max: 0,
  String id,
  String reason,
}) {
  actionDispatcher.setDispatcher(expectAsync1((Action<dynamic> action) {
    if (verifier != null) verifier(action as Action<T>);
    if (verfier != null) verfier(action as Action<T>);
  }, count: count, max: max, id: id, reason: reason));
}
```